src/marks/waffle.js

```diff
@@ -35,6 +35,7 @@ export class WaffleY extends BarY {
 
 function waffleRender(y) {
   return function (index, scales, values, dimensions, context) {
+    const {ariaLabel, href, title, ...visualValues} = values;
```

Comment:
If this needs to be done somewhere else in the future, we should probably instead split `applyChannelStyles` into two functions (`applyChannelStyles` and `applyChannelSemantics`).